qutebrowser/app.py

```diff
@@ -404,6 +405,13 @@ def _init_modules(args, crash_handler):
     config.init(qApp)
     save_manager.init_autosave()
 
+    log.init.debug("Initializing keys...")
+    keyconf.init(qApp)
+    save_manager.init_autosave()
```

Comment:
nope, good catch. Actually, this came from 5734c469b which isn't necessary as I solved the circular import another way, but it might not be bad to eliminate `config`s dependence on `keyconf` anyways (it only depended on the module to initialize it)